src/community/ogcapi/dggs/ogcapi-dggs/src/main/java/org/geoserver/ogcapi/v1/dggs/DGGSService.java

```diff
@@ -578,6 +578,7 @@ protected String getItemsPath(String collectionName) {
         return new FeaturesResponse(request.getAdaptee(), response);
     }
 
+    @Deprecated
```

Comment:
Javadoc or comment to explain the deprecation?

Reply:
Never mind, removed deprecation, leftover from an IDE suggestion.